notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1345_TEA_Vibration_audio_freq_analysis.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    raw = open_wave('Audio_'+n+'.wav','rb')
</pre><p>I suggest to format the string using "f" and "{   }"</p><p>raw = open_wave(f'Audio_{n}.wav','rb')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
ok !